examples/llm/mixture_of_agents.yaml

```diff
@@ -193,6 +193,7 @@ flows:
       # Vertex AI LLM Request
       - component_name: llm_request
         component_module: langchain_chat_model
+        num_instances: 3
```

Comment:
Just checking that you intended to commit this change.

Reply:
Yes, it was creating a bottle neck when you'd send multiple events (also considering that each event would get duplicated to 3 layers)

I also thought it would be a nice example to show how they users can use the num_instances option